src/css/style.css

```diff
@@ -446,11 +445,22 @@ button {
     text-transform: none;
     line-height: initial;
 }
-/* Special handling for the welcome icon that does not have an icon */
+/* Special handling for the welcome and congratulations icons that do not have an icon */
 .gpii-fd-navIcon-welcome {
     display: inline-block;
     margin: 0.2rem -0.3rem 0 0;
 }
+.gpii-fd-navIcon-congratulations {
```

Comment:
The definitions for .gpii-fd-navIcon-welcome and .gpii-fd-navIcon-congratulations are same and can be combined into one.